src/main/resources/application.properties

```diff
@@ -1,5 +1,9 @@
-spring.h2.console.enabled=true
-spring.datasource.url=jdbc:h2:mem:mydb
-spring.jpa.properties.hibernate.show_sql=true
-spring.jpa.properties.hibernate.format_sql=true
+
+spring.datasource.url=jdbc:mysql://localhost:3306/electroshop?serverTimezone=UTC
```

Comment:
Tutaj to UTC+2 powinno być na końcu?

Reply:
Na czacie fb napisałem ci UTC+2, ale jednak okazało się że ma być UTC, bo jeszcze wywalało co innego